diag_manager/fms_diag_object.F90

```diff
@@ -29,14 +29,16 @@ module fms_diag_object_mod
 use fms_diag_file_object_mod, only: fmsDiagFileContainer_type, fmsDiagFile_type, fms_diag_files_object_init
 use fms_diag_field_object_mod, only: fmsDiagField_type, fms_diag_fields_object_init, get_default_missing_value
 use fms_diag_yaml_mod, only: diag_yaml_object_init, diag_yaml_object_end, find_diag_field, &
-                           & get_diag_files_id, diag_yaml, get_diag_field_ids, DiagYamlFilesVar_type
+                           & get_diag_files_id, diag_yaml, diagYamlFilesVar_type, get_diag_field_ids
```

Comment:
Why is this changed?

Reply:
caused by merging locally; will be reverted.